networks/p2p/server.go

```diff
@@ -577,8 +577,9 @@ func (srv *MultiChannelServer) setupConn(c *conn, flags connFlag, dialDest *disc
 			dialDest.TCPs = append(dialDest.TCPs, uint16(listenPort))
 		}
 		srv.AddPeer(dialDest)
-		logger.Info("[Dial] Try to update dial candidate with a multichannel peer", "id", dialDest.ID, "addr", dialDest.IP, "port", dialDest.TCPs)
-		return nil
+		logger.Info("[Dial] update and retry the dial candidate as a multichannel",
+			"id", dialDest.ID, "addr", dialDest.IP, "port", dialDest.TCPs)
+		return errUpdateDial
```

Comment:
@winnie-byun If https://github.com/klaytn/klaytn/pull/811 is merged, this code will be reached because `len(dialDest.TCPs` is always greater than 0, right?

Reply:
You are right. I will change the condition in PR#811. Thank you.